Mandriva has the drivers in the kernel, so we don't want to build
them. Disable checking for kernel sources when they aren't present.

diff -p -up linux-wlan-ng-0.2.8/Configure.orig linux-wlan-ng-0.2.8/Configure
--- linux-wlan-ng-0.2.8/Configure.orig	2007-05-07 19:00:22.000000000 -0300
+++ linux-wlan-ng-0.2.8/Configure	2008-08-20 13:57:10.000000000 -0300
@@ -229,6 +229,8 @@ if [ "$LINUX_SRC" = "" ] ; then
     LINUX_SRC=/lib/modules/$CUR_RELEASE/build
 fi
 
+if [ -d "$LINUX_SRC" ]; then
+
 ask_str "Linux source directory" LINUX_SRC
 
 if [ ! -f $LINUX_SRC/include/linux/version.h ] ; then
@@ -354,6 +356,8 @@ $ECHO ""
 	dump_str "KERNEL_CFLAGS=$KERNEL_CFLAGS"
 	dump_str "KERNEL_MODFLAGS=$KERNEL_MODFLAGS"
 
+fi
+
 #=======================================================================
 # If compiling for pcmcia-cs, find the PCMCIA source tree
 
